### PR TITLE
fix(NotificationListNew): remove top pattern

### PR DESCRIPTION
**Pull request by saghdaey**
### Overview

Tuba pointed out that the top pattern line was actually supposed to be static and below the title 

### PR Checklist

- [ ] Related to designs:
- [x] Related to JIRA ticket: EGG-985
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1183,6 +1183,13 @@
   dependencies:
     "@emotion/memoize" "^0.7.4"
 
+"@emotion/is-prop-valid@^1.1.0":
```
Comment (saghdaey) [inline yarn.lock:1186]: I ran yarn locally which caused these changes in yarn.lock, is that fine? @JoshuaKGoldberg @codecaaron

Comment (codecaaron) [reply yarn.lock:1186]: Oh whoops thats me

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.2.43-alpha.592a3e.0<br/>@codecademy/gamut-labs@13.2.6-alpha.592a3e.0<br/>@codecademy/gamut@29.5.1-alpha.592a3e.0<br/>@codecademy/styleguide@33.4.1-alpha.592a3e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60638d527734a434c7ad992c--gamut-preview.netlify.app>https://60638d527734a434c7ad992c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60638d527734a434c7ad992c>Deploy Logs</a></p>

**Review by dreamwasp — APPROVED** [review]

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.2.43-alpha.ec3b3b.0<br/>@codecademy/gamut-labs@13.2.6-alpha.ec3b3b.0<br/>@codecademy/gamut@29.5.1-alpha.ec3b3b.0<br/>@codecademy/styleguide@33.4.1-alpha.ec3b3b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://606390578f34f12e2ccdb6eb--gamut-preview.netlify.app>https://606390578f34f12e2ccdb6eb--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/606390578f34f12e2ccdb6eb>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.2.44-alpha.908c85.0<br/>@codecademy/gamut-labs@13.2.7-alpha.908c85.0<br/>@codecademy/gamut@29.5.2-alpha.908c85.0<br/>@codecademy/styleguide@33.4.2-alpha.908c85.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6063ae28ed188400a852debe--gamut-preview.netlify.app>https://6063ae28ed188400a852debe--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6063ae28ed188400a852debe>Deploy Logs</a></p>

**Pull request merged**